libpod/events/logfile.go

```diff
@@ -19,6 +19,11 @@ type EventLogFile struct {
 
 // Writes to the log file
 func (e EventLogFile) Write(ee Event) error {
+	eventJSONString, err := ee.ToJSONString()
```

Comment:
I don't think this change should be in this PR?

Reply:
Definitely not. Thanks for checking!